numba/unicode.py

```diff
@@ -1286,6 +1286,31 @@ def impl(a):
     return impl
 
 
+@overload_method(types.UnicodeType, 'istitle')
+def unicode_istitle(s):
+    """
+    Implements UnicodeType.istitle()
+    """
+
+    def impl(s):
+        str_has_abc = False
```

Comment:
What's `_abc` mean here? also this alg looks a lot like something from CPython, is a citation with SHA missing?

Reply:
Hi,
By "str_has_abc" I have tried to say that string has alphabetical characters.
The "cased" from CPython algorithm does not express its meaning for me.
Ok, I will add the link to CPython here

Reply:
I think the `cased`/`previous_is_cased` is an abbreviated form of `is_title_cased`/`previous_is_title_cased`, this algorithm is essentially looking for certain patterns in a string that may occur between title-cased and uncased/lower-case characters.

Reply:
Ok, I understand.
Then I revert names back.

Reply:
Great, thanks.